### PR TITLE
Convert to Class and change config

### LINKED ISSUE #17: Create Bot Application
Model off [SourceQuerBot](https://github.com/WardPearce/SourceQueryBot)

### LINKED ISSUE #31: Convert Bot to Class
- [x] Convert Bot to a class file
- [x] Create runner file that loads the information into the bot

### PR DESCRIPTION
**Pull request by yannickgloster**
- Changed the bot to be through a class and have a runner file
- Changed config to start support for multiple servers

Closes #31 #17

### CODE REVIEW

File: cogs/csgo.py [hunk 1]
```diff
@@ -215,10 +215,12 @@ async def pug(self, ctx):
         match_config_json = await ctx.send(file=discord.File('match_config.json', '../match_config.json'))
         await ctx.send('If you are coaching, once you join the server, type .coach')
         await asyncio.sleep(0.3)
-        valve.rcon.execute(self.bot.server_address, self.bot.RCON_password, 'exec triggers/get5')
+        valve.rcon.execute((self.bot.servers[0]["server_address"], self.bot.servers[0]["server_password"]),
+                           self.bot.servers[0]["RCON_password"], 'exec triggers/get5')
```
Comment (lexesj) [inline cogs/csgo.py:219]: Typo, should be `server_password` should be `server_port`

File: cogs/csgo.py [hunk 2]
```diff
@@ -215,10 +215,12 @@ async def pug(self, ctx):
...
         await self.connect(ctx)
         await asyncio.sleep(10)
-        valve.rcon.execute(self.bot.server_address, self.bot.RCON_password,
+        valve.rcon.execute((self.bot.servers[0]["server_address"], self.bot.servers[0]["server_password"]),
+                           self.bot.servers[0]["RCON_password"],
```
Comment (lexesj) [inline cogs/csgo.py:223]: Typo, same as above

File: cogs/setup.py [hunk 3]
```diff
@@ -78,7 +42,8 @@ async def link_error(self, ctx, error):
                       brief='Sends a message to the server to test RCON', usage='<message>')
     @commands.has_permissions(administrator=True)
     async def RCON_message(self, ctx, *, message):
-        test = valve.rcon.execute(self.bot.server_address, self.bot.RCON_password, f'say {message}')
+        test = valve.rcon.execute((self.bot.servers[0]["server_address"], self.bot.servers[0]["server_password"]),
+                           self.bot.servers[0]["RCON_password"], f'say {message}')
```
Comment (lexesj) [inline cogs/setup.py:46]: Typo, same as above

File: cogs/setup.py [hunk 4]
```diff
@@ -92,9 +57,9 @@ async def RCON_message_error(self, ctx, error):
     @commands.command(help='This command unbans everyone on the server. Useful fix',
                       brief='Unbans everyone from the server', hidden=True)
     @commands.has_permissions(administrator=True)
-
     async def RCON_unban(self, ctx):
-        unban = valve.rcon.execute(self.bot.server_address, self.bot.RCON_password, 'removeallids')
+        unban = valve.rcon.execute((self.bot.servers[0]["server_address"], self.bot.servers[0]["server_password"]),
+                           self.bot.servers[0]["RCON_password"], 'removeallids')
```
Comment (lexesj) [inline cogs/setup.py:62]: Typo, same as above

**Review by lexesj — CHANGES_REQUESTED** [review]

File: cogs/csgo.py (hunk 1 above)
Comment (yannickgloster) [reply cogs/csgo.py:219]: Fixed in latest commit

File: cogs/setup.py (hunk 3 above)
Comment (yannickgloster) [reply cogs/setup.py:46]: Fixed in Commit

File: cogs/csgo.py (hunk 2 above)
Comment (yannickgloster) [reply cogs/csgo.py:223]: Fixed in Commit

File: cogs/setup.py (hunk 4 above)
Comment (yannickgloster) [reply cogs/setup.py:62]: Fixed in Commit

**Review by lexesj — APPROVED** [review]
Looks good!

**Pull request merged**